site/content/docs/CICD/vercel.md

```diff
@@ -12,54 +12,50 @@ aliases:
 
 The Vercel integration will help you by:
 
-1. Automatically creating a pre-scripted browser check to catch any errors and failed requests as your web page loads.
-2. Enabling you to run checks against preview and production deployments on Vercel.
+-  Automatically creating a pre-scripted browser check to catch any errors and failed requests as your web page loads.
+-  Enabling you to run checks against preview and production deployments on Vercel.
 
 > Using Vercel deployment protection? Read [this section](/docs/cicd/vercel-deployment-protection/) on how to make this work with Checkly.
 
-## Installation
-
-To install Checkly's native Vercel integration, navigate to `Integrations`, under your account's dropdown menu.
-
-![vercel integration step 1](/docs/images/cicd/vercel/vercel_step1a.png)
-
-After scrolling down, click the `Vercel marketplace` button.
-
-![vercel integration step 2](/docs/images/cicd/vercel/vercel_step2.png)
+## Installing the native integration
 
-On the marketplace page for Checkly, click `Add Integration`.
+To install Checkly's managed Vercel integration, navigate to the [integration's marketplace listing](https://vercel.com/integrations/checkly) and click `Install`.
 
-![vercel integration step 3](/docs/images/cicd/vercel/vercel_step3.png)
+1. Select `Create new account` in the installation guide and click `Next` to accept the terms and conditions.
+2. Select your plan type. You can read about our pricing plans [here](https://www.checklyhq.com/pricing/), and more about how we bill checks [here](/docs/monitoring/check-pricing/).
+3. Name your account and review your plan selection. You can change your account name later.
+4. Press `Create` to continue to the integration settings page. Select `Getting started` to go to the setup guide in Vercel. You can also click `Open in Checkly` to go through the Checkly + Vercel onboarding and get you started with an example project showcasing how to use Monitoring as Code together with Vercel. You can also jump straight to our [example NextJS project](https://github.com/checkly/nextjs-checkly-starter-template) which includes a guide on how to use Monitoring as Code with a NextJS project.
+5. If you want to directly connect your Checkly checks with Vercel and run them against preview and deployments on Vercel, continue with installing the connected integration, as explained below.
 
-Next, follow the installation wizard to grant the integration access to the right Vercel scope and projects.
+## Installing the connected integration
 
-![vercel integration step 4](/docs/images/cicd/vercel/vercel_step4.png)
+1. To install Checkly's connected Vercel integration, navigate to `Integrations`, under your account's dropdown menu and click the `Vercel marketplace` button, or go directly to the [integration listing on the Vercel marketplace](https://vercel.com/integrations/checkly).
```

Comment:
can we add a link here to https://app.checklyhq.com/settings/account/integrations